spec/controllers/users/phone_confirmation_controller_spec.rb

```diff
@@ -42,14 +42,17 @@
         end
 
         it 're-sends existing code' do
-          expect(SmsOtpSenderJob).to receive(:perform_later).
-            with(
-              code: '1234',
-              phone: '+1 (555) 555-5555',
-              otp_created_at: controller.current_user.direct_otp_sent_at.to_s
-            )
-
-          get :send_code
+          now = Time.zone.now
+          Timecop.freeze(now) do
+            expect(SmsOtpSenderJob).to receive(:perform_later).
+              with(
+                code: '1234',
+                phone: '+1 (555) 555-5555',
+                otp_created_at: now.to_s
+              )
+
+            get :send_code
+          end
```

Comment:
I haven't thought through the ideal solution yet, but I wanted to make a note that it would be preferable to have tests that don't depend on implementation. These tests will break after my refactor.